opentelemetry/proto/collector/logs/v1/logs_service.proto

```diff
@@ -43,4 +43,35 @@ message ExportLogsServiceRequest {
 }
 
 message ExportLogsServiceResponse {
+  // The details of a partially successful export request.
+  //
+  // If the request is only partially accepted
+  // (i.e. when the server accepts only parts of the data and rejects the rest)
+  // the server MUST initialize the `partial_success` field and MUST
+  // set the `rejected_log_records` with the number of log records it rejected.
+  //
+  // The server MUST NOT set the `partial_success` field when the
+  // request was accepted in its entirety.
+  ExportLogsPartialSuccess partial_success = 1;
+}
+
+message ExportLogsPartialSuccess {
+  // The number of rejected log records.
+  //
+  // If the request is only partially accepted, the server MUST set
```

Comment:
>If the request is only partially accepted

Isn't this part superfluous? If the request is fully accepted the entire ExportLogsPartialSuccess should not exist. This is what the line 48 says.

Reply:
I had the same thought but ended up leaving just to be "extra clear" on the expectations. But I can remove, I agree it's duplicated.

Reply:
Removed in [32ab10e](https://github.com/open-telemetry/opentelemetry-proto/pull/414/commits/32ab10ec580906780e6cedb47f362c8d35be83f1)